### ISSUE TITLE
Cannot start docker container

### ISSUE THREAD

**Issue by Relfos** [slipo/neo-scan-docker#3]
After running docker-compose up and waiting a few minutes, I get this error

```
neo-scan       | warning: function NeoscanSync.HttpCalls.url/1 is undefined (module NeoscanSync.HttpCalls is not available)
neo-scan       | Found at 4 locations:
neo-scan       |   lib/neoscan/chain_assets/assets.ex:211
neo-scan       |   lib/neoscan/repair/repair.ex:42
neo-scan       |   lib/neoscan/repair/repair.ex:169
neo-scan       |   lib/neoscan/repair/repair.ex:180
neo-scan       |
neo-scan       | Generated neoscan app
neo-scan       | ==> neoprice
neo-scan       | Compiling 8 files (.ex)
neo-scan       | Generated neoprice app
neo-scan       | ==> neoscan_monitor
neo-scan       | Compiling 6 files (.ex)
neo-scan       | warning: variable "broadcast" is unused
neo-scan       |   lib/neoscan_monitor/monitor/utils.ex:10
neo-scan       |
neo-scan       | warning: function NeoscanSync.Blockchain.get_block_by_height/2 is undefined (module NeoscanSync.Blockchain is not available)
neo-scan       |   lib/neoscan_monitor/monitor/utils.ex:66
neo-scan       |
neo-scan       | warning: function NeoscanSync.Blockchain.get_current_height/1 is undefined (module NeoscanSync.Blockchain is not available)
neo-scan       |   lib/neoscan_monitor/monitor/utils.ex:17
neo-scan       |
neo-scan       | Generated neoscan_monitor app
neo-scan       | ==> neoscan_sync
neo-scan       | Compiling 6 files (.ex)
neo-scan       | Generated neoscan_sync app
neo-scan       | ==> neoscan_web
neo-scan       | Compiling 36 files (.ex)
neo-scan       | Generated neoscan_web app
neo-scan       |
neo-scan       | 17:23:20.904 [error] GenServer #PID<0.4767.0> terminating
neo-scan       | ** (ArgumentError) argument error
neo-scan       |     (postgrex) lib/postgrex/utils.ex:67: anonymous fn/1 in Postgrex.Utils.parse_version/1
neo-scan       |     (elixir) lib/enum.ex:1294: Enum."-map/2-lists^map/1-0-"/2
neo-scan       |     (elixir) lib/enum.ex:1294: Enum."-map/2-lists^map/1-0-"/2
neo-scan       |     (postgrex) lib/postgrex/utils.ex:67: Postgrex.Utils.parse_version/1
neo-scan       |     (postgrex) lib/postgrex/protocol.ex:641: Postgrex.Protocol.bootstrap_send/6
neo-scan       |     (postgrex) lib/postgrex/protocol.ex:475: Postgrex.Protocol.handshake/2
neo-scan       |     (db_connection) lib/db_connection/connection.ex:134: DBConnection.Connection.connect/2
neo-scan       |     (connection) lib/connection.ex:622: Connection.enter_connect/5
neo-scan       |     (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
neo-scan       | Last message: nil
neo-scan       | State: Postgrex.Protocol
neo-scan       | ** (Mix) The database for Neoscan.Repo couldn't be created: an exception was raised:
neo-scan       |     ** (ArgumentError) argument error
neo-scan       |         (postgrex) lib/postgrex/utils.ex:67: anonymous fn/1 in Postgrex.Utils.parse_version/1
neo-scan       |         (elixir) lib/enum.ex:1294: Enum."-map/2-lists^map/1-0-"/2
neo-scan       |         (elixir) lib/enum.ex:1294: Enum."-map/2-lists^map/1-0-"/2
neo-scan       |         (postgrex) lib/postgrex/utils.ex:67: Postgrex.Utils.parse_version/1
neo-scan       |         (postgrex) lib/postgrex/protocol.ex:641: Postgrex.Protocol.bootstrap_send/6
neo-scan       |         (postgrex) lib/postgrex/protocol.ex:475: Postgrex.Protocol.handshake/2
neo-scan       |         (db_connection) lib/db_connection/connection.ex:134: DBConnection.Connection.connect/2
neo-scan       |         (connection) lib/connection.ex:622: Connection.enter_connect/5
neo-scan       |         (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
neo-scan exited with code 1
```

**Comment by slipo** [comment]
@Relfos does it look related to this? https://github.com/slipo/neo-scan-docker/issues/2

Very strange. It was working for me. I'm going to `docker-compose pull` and see if the postgres package broke somehow.

**Comment by Relfos** [comment]
Yes, it is related, or in fact, seems to be same exact problem, sorry for opening it, I had a quick look at existing issues but missed that.

**Comment by Relfos** [comment]
Thank you, it is now fixed by [#4](https://github.com/slipo/neo-scan-docker/pull/4).
